code/model/DMSDocument.php

```diff
@@ -145,6 +146,14 @@ public function canDelete($member = null)
         return $this->canView();
     }
 
+    /**
+     * Allow files to be opened in a browser as the default value. This can be superceded by
+     * values set on a per-document basis.
+     * @static
+     */
+    public static function set_default_download_behaviour( $behaviour ){
```

Comment:
There's no need for a setter, we can just tell people to use the config directly. We should recommend setting in yaml primarily.